shm/inc/Cargo.hpp

```diff
@@ -2,17 +2,19 @@
 
 #include <string>
 
+//class resposible for manage commodity in game
 class Cargo {
 public:
-    Cargo(const std::string&, size_t, size_t);
+    //constructs an object of cargo in game
```

Comment:
Im not sure about this comment, probably everyone knows what the constructor is for :)

Reply:
yes, i forgot to delet it :-)